.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -63,6 +64,14 @@ jobs:
         fi
 
         go run ./scanner/cmd/updater -output-dir=${{ env.ROX_PRODUCT_VERSION }}
+
+        # The vulnerability file created by older versions can remain the name vulns.json.zst in the GCS bucket.
+        for file in output.json.{zst,ztd}; do
+          if [ -f "${{ env.ROX_PRODUCT_VERSION }}/${file}" ]; then
+            mv "${{ env.ROX_PRODUCT_VERSION }}/${file}" "${{ env.ROX_PRODUCT_VERSION }}/vulns.json.zst"
+            break
+          fi
+        done
```

Comment:
nit: This necessary? I don't believe any of the 'Scanner V2' versions use this file structure / extensions - unless I'm missing something - assuming all the Scanner V4 'components' should now be expecting the same file name(s).

Reply:
.github/workflows/scanner-release-vuln-update.yaml only creates v4 bundles since 4.2.0.
There are old version such as 4.3.0 that publishes output.json.ztd. Recently we changed v4 vuln bundle name from output.json.zst to **_vulns.json.zst_** so we need to make sure all versions have the same file name
https://github.com/stackrox/stackrox/pull/9688

Reply:
As for now, I will probably still need the 4.3.x stuff for testing. Maybe remove those temporary files after 4.4 release?

Reply:
Sure sounds good.